client/src/routes/user/ListPage.tsx

```diff
@@ -99,6 +102,21 @@ export function UserListPage() {
                     />
                 </>
             )}
+            <Divider sx={{ mt: "60px", mb: "40px" }} />
+            <Paper sx={{
+                mb: 4,
+                pt: 2,
+                pb: 1,
+                px: 2
+            }}>
+                <Grid container  spacing={0}>
+                    <Grid item xs={12}><h4 style={{margin: "0px"}}>Nedlast e-postlister</h4></Grid>
+                    <EmailLink users={actualUsers} label="Alle"/>
+                    <EmailLink users={activeUsers} label="Aktive"/>
+                    <EmailLink users={boardUsers} label="Styret"/>
```

Comment:
Ikke nødvendig å ha liste for styremailer. Hvis man skal sende mail til alle i styret skal man sende til `styret@motstanden.no`